openproblems/tasks/chromatin_potential/methods/beta.py

```diff
@@ -69,33 +87,38 @@ def _get_annotation(adata, retries=3):
     )
 
 
-def _atac_genes_score(adata, top_genes=500, threshold=1):
+def _atac_genes_score(adata, top_genes=2000, threshold=1, method="beta"):
     """Calculate gene scores and insert into .obsm."""
     import pybedtools
 
     # get annotation for TSS
     _get_annotation(adata)
     # basic quality control
-    if adata.shape[0] >= 600:
+    if adata.shape[0] >= 10000:
         sc.pp.filter_cells(adata, min_genes=200)
         sc.pp.filter_genes(adata, min_cells=5)
 
     adata = adata[:, ~pd.isnull(adata.var.loc[:, "chr"])].copy()
-    adata.var["mt"] = adata.var.gene_short_name.str.lower().str.startswith(
-        "mt-"
-    )  # annotate the group of mitochondrial genes as 'mt'
+    if adata.uns["species"] == "mus_musculus":
+        adata.var["mt"] = adata.var.gene_short_name.str.lower().str.startswith(
+            "mt-"
+        )  # annotate the group of mitochondrial genes as 'mt'
+    else:
```

Comment:
This is too broad. Can you specify human here? Which other species use "MT-"? What do we do if it's neither human or mouse?

Reply:
This should work for both human and mouse, we only have benchmark datasets from these two species so far. For other species, I am not sure if they are using "MT-". I'll add a condition here